modules/bot.py

```diff
@@ -983,42 +1034,44 @@ def pick_player(self, member, args):
             return
 
         if len(args):
-            targetid = args[0].lstrip("<@!").rstrip(">")
```

Comment:
A nitpick that bothered me - bad variable naming that concatenates two words into one.